demos/polls/README.rst

```diff
@@ -7,20 +7,57 @@ similar to django one.
 Installation
 ============
 
-Install the app::
+Prepare development environment
+-------------------------------
+
+We suggest you to start by creating an isolated Python working environment::
+
+    $ python3 -m venv env
+    $ source env/bin/activate
+
+
+Install the app and it's requirements::
 
     $ cd demos/polls
     $ pip install -e .
 
-Create database for your project::
 
-    bash sql/install.sh
+Prepare database
+----------------
+Install Postgresql database server: http://www.postgresql.org/download/
+(to use postgres in more isolated way you may also use docker. We will explain how to do that
+after environment variables setup.)
+
+Create ``.env`` file in ``polls`` folder with db related configuration values. It is a common practice to not
+hardcode such settings because they differ between e.g. prod and dev environments::
+
+    DB_HOST=localhost
+    DB_PORT=5432
+    DB_NAME=aiohttpdemo_polls
+    DB_USER=aiohttpdemo_user
+    DB_PASS=aiohttpdemo_user
+
+Export these variables (to be able to use them inside db initializing script)::
 
+    $ source .env
```

Comment:
Better use smth like this
```python
import envparse
envparse.Env.read()
```
instead of tutoring them do shell scripting.

Reply:
- afaik envparse is not built in package. 
- Yes, It would be nice to protect user from all this cruft, but in this PR I just change minimum in existing sh's to allow parameterization. 
- My next goal is to make docker-compose with necessary variables and make it primary setup method.

Reply:
I believe much more people knows about how to enter into virtualenv via `source` rather than what is `envparse`.  Also, it's not good to embed such things into the code.

Reply:
Fair enough